ports/expat/portfile.cmake

```
@@ -27,16 +28,11 @@ vcpkg_configure_cmake(
 vcpkg_install_cmake()
 
 vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/expat)
```

```suggestion
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/expat-2.2.9)
```
since version isn't use anywhere else